### DISCUSSION TITLE
interim release?

### DISCUSSION THREAD

**Discussion by steevhise** [paulmillr/chokidar#1338] [General]
I know you're working on v4 but any chance of publishing a 3.6.1 in the meantime? There's a security issue with braces, which chokidar stopped using several commits ago but this hasn't made it to a release.
https://github.com/advisories/GHSA-grv7-fg5c-xmjg

**Comment by paulmillr** [comment]
1. Dep version ranges are used. Google what it does.
2. It's not really a security issue. It's been inflated by security researchers.
